packages/backend/src/apps/vault-workspace/actions/create-row/index.ts

```diff
@@ -28,16 +28,28 @@ export default defineAction({
         'Put a comma between each value. Enclose values with double-quotes (") if you think it may contain commas (e.g. form answers). Values CANNOT contain double quotes.',
     },
   ],
+  preprocessVariable: (key, value) => {
+    if (key !== 'values' && key !== 'columns') {
+      return value
+    }
+    if (typeof value !== 'string') {
+      return value
+    }
```

Comment:
actually i thought that there are only two keys for this action here, when would this be called ah?

Reply:
I think this one more for defensive coding.... But tbh i think this will only ever be called with `values` unless we change something drastically.